yt_dlp/extractor/line.py

```diff
@@ -82,14 +83,16 @@ def _real_extract(self, url):
 
         # like_count requires an additional API request https://tv.line.me/api/likeit/getCount
 
+        thumbnail = try_get(video_info, lambda x: x['meta']['cover']['source']) or []
+        thumbnail = thumbnail.split('?')[0]#Full Size Thumbnail
+
         return {
             'id': video_id,
             'title': title,
             'formats': formats,
             'extra_param_to_segment_url': extra_query[1:],
             'duration': duration,
-            'thumbnails': [{'url': thumbnail['source']}
-                           for thumbnail in video_info.get('thumbnails', {}).get('list', [])],
```

Comment:
do they no longer exist?

Reply:
They are fast seeking images and not thumbnails.

Reply:
it's okay to still have them imo.
I'm not sure if thumbnails and thumbnail both can be returned at the same time....if not then just append your thumbnail to this (looks like it won't be inline anymore)

Reply:
They are not usable thumbnails, they are just 100px images from spaced out time codes.